snap/validate.go

```diff
@@ -161,6 +161,13 @@ func validateField(name, cont string, whitelist *regexp.Regexp) error {
 	return nil
 }
 
+func validateAppSocket(name string, socket *SocketInfo) error {
+	if socket.ListenStream == "" {
+		return fmt.Errorf("app socket '%s' has empty ListenStream", name)
```

Comment:
`("socket %q must define listen-stream", name)`

The name verification mentioned earlier may be performed here.

Reply:
Done